windows-installer/template-nsi.patch

```diff
@@ -0,0 +1,183 @@
+--- pyapp.nsi	2020-03-31 01:46:13.118535700 +0200
```

Comment:
How is a good old `diff`. What its usage becomes less clear now. Initally I wanted to keep track of all modifications done to the original file, but I realize that you can get them if you keep track of file source somewhere, and it is in the inline comments at the beginning of `template.nsi`.

So I suppose it is not useful anymore. I remove it.

Reply:
I was not clear enough in my previous message: I wanted to keep track of modifications to dynamically change the original file during the installer compilation, but dropped the idea because it was too complex. So now it is useless.